platform/ios/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ Mapbox welcomes participation and contributions from everyone. Please read [CONT
 * Added an MGLStyle.localizesLabels property, off by default, that localizes any Mapbox Streets–sourced symbol layer into the user’s preferred language. ([#9582](https://github.com/mapbox/mapbox-gl-native/pull/9582))
 * Added an additional camera method to MGLMapView that accepts an edge padding parameter. ([#9651](https://github.com/mapbox/mapbox-gl-native/pull/9651))
 * Fixed an issue with the scaling of the user location annotation’s horizontal accuracy indicator. ([#9721](https://github.com/mapbox/mapbox-gl-native/pull/9721))
+* Fixed an issue that caused querying features in a source added through Mapbox Studio returned nil. ([#9784](https://github.com/mapbox/mapbox-gl-native/pull/9784))
```

Comment:
Nit: s/returned/to return/. Also, if I understand correctly, the issue affected all vector sources, not just those added through Mapbox Studio.